hardhat.config.js

```diff
@@ -90,19 +96,21 @@ module.exports = {
     '*': {
       'code-size': withOptimizations,
       'unused-param': !argv.coverage, // coverage causes unused-param warnings
+      'transient-storage': false,
```

Comment:
I couldn't find this option in the docs, where's it coming from?

Reply:
its a new one introduced in 0.8.24, I had to make a PR to add this https://github.com/frangio/hardhat-ignore-warnings/pull/6/files